service/templates/admin.html

```diff
@@ -2570,9 +2571,70 @@ <h2 class="modal-title">[[modalTitle]]</h2>
 								</md-card-content>
 							</md-card>
 						</section>
-
-					</md-card-content>
-
+						<section layout="row" flex>
+							<md-card flex=50 id="project_creation" style="max-width:49%">
+								<md-card-title>
+									<md-card-title-text>
+										<span>Project Creation Settings</span>
+									</md-card-title-text>
+									<md-button ng-click="clearSection('project_creation')">
+										<i class="fa fa-trash"></i>
+									</md-button>
+								</md-card-title>
+								<md-card-content>
+									<section layout="row" flex>
+										<md-input-container class="control" style="width:45%;">
+											<label>Total Project Limit</label>
+											<input type="number" ng-model="Settings.project_creation.total_project_limit">
+										</md-input-container>
+										<md-input-container class="control" style="width:45%;">
+											<label>Project Per Repo Limit</label>
+											<input type="number" ng-model="Settings.project_creation.repo_project_limit">
+										</md-input-container>
+									</section>
+										<md-card style="margin-bottom:20px;">
+											<md-card-title>
+												<md-card-title-text>
+													<span>Repos to Override</span>
```

Comment:
Can we also change this to Repo Exceptions? (and related variables)

Reply:
sorry i have no idea why i didnt change those in the last comment 
updated